src/main/java/mapWithNameAndPassword/CustomerNameAndPass.java

```diff
@@ -0,0 +1,34 @@
+package mapWithNameAndPassword;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Temporary class for store customers names and passwords
+ */
+public class CustomerNameAndPass {
```

Comment:
refactor the name of the class, try to avoid redundant words in naming like "and"

Reply:
will suit the name CustomerNamePass?